src/java/org/apache/cassandra/tools/nodetool/Sjk.java

```diff
@@ -143,7 +145,7 @@ else if (isListCommands())
                 {
                     for (String cmd : commands.keySet())
                     {
-                        System.out.println(String.format("%8s - %s", cmd, parser.getCommandDescription(cmd)));
+                        outStream.println(String.format("%8s - %s", cmd, parser.getCommandDescription(cmd)));
```

Comment:
in the catch block for `CommandAbortedError` there are other logging commands, can replace them as well.